trpc/router/goal.ts

```diff
@@ -672,6 +672,7 @@ export const goal = router({
 
         if (!commentAuthor) return null;
         if (!actualGoal) return null;
+        if (!input.goalId) return null;
```

Comment:
hm, I think this is useless check in this place 'cause `input.goalId` early uses for query from a db 😄 
or you need move this check before query data